Organizing Top Menu and adding 2021 Hack Week info (#9)
PlasmaPy/plasma-hack-week: pages/2021/about.md

title: About 2021 Hack Week
hidetitle: True

# Plasma Hack Week 2021

> Welcome to the inaugural **Plasma Hack Week**!!  Our first even will be held

Reviewer: ```suggestion
Welcome to the inaugural **Plasma Hack Week**!!  Our first event will be held
```

remotely from June 28 - July 2, 2021.  In the week before the event we will

> hold prequel tutorialson tpics such as Python, git & GitHub, contributing to

Reviewer: ```suggestion
hold prequel tutorials on topics such as Python, git & GitHub, contributing to
```

title: About 2021 Hack Week
hidetitle: True

# Plasma Hack Week 2021

Welcome to the inaugural **Plasma Hack Week**!!  Our first event will be held 
remotely from June 28 - July 2, 2021.  The event is **free** and sets out to 
increase awareness of open-source software for the Plasma Community:
the benefits it can have on the community, what is currently available, and
how to foster community development of open-source software.  As such, topics will
range form demonstrating and using current open-source plasma software projects

> to software engineering skills.  These topics will be present through both

Reviewer 2: ```suggestion
to software engineering skills.  These topics will be presented through both
```

> to software engineering skills.  These topics will be present through both

Reviewer 2: ```suggestion
to software engineering skills.  These topics will be presented through both
```

structured learning activities, such as guided tutorials, and unstructured project
time for collaborative learning.

> In the week leading up to the **Plsama Hack Week**, a few more fundamentally

Author: ```suggestion
In the week leading up to the **Plasma Hack Week**, a few more fundamentally
```
